gatling-load-testing/src/gatling/simulations/BrowseAndListenSimulation.scala

```diff
@@ -5,98 +5,80 @@ import io.gatling.http.Predef._
 
 class BrowseAndListenSimulation extends Simulation {
 
-  val frontendUrl = "https://wayland-prisoner-content-hub-staging.apps.live-1.cloud-platform.service.justice.gov.uk"
-  val backendUrl = "https://cms-prisoner-content-hub-staging.apps.live-1.cloud-platform.service.justice.gov.uk"
+  val frontendUrl = sys.env.getOrElse("FRONTEND_URL", "https://wayland-prisoner-content-hub-staging.apps.live-1.cloud-platform.service.justice.gov.uk")
```

Comment:
Do we want these to be something that might mean the tests pass? Just thinking if the env vars weren't set we wouldn't know about it as the tests would still pass with these defaults?

Reply:
100% on this, removed this configuration and expanded the setUp